.eslintrc

```diff
@@ -39,7 +39,7 @@
     "space-before-function-paren": [2, {
       "anonymous": "always", "named": "never"
     }],
-    "spaced-line-comment": [2, "always"],
+    "spaced-comment": [2, "always"],
```

Comment:
I took all of your rules and only added a few new. You can [take a look at the result](https://gist.github.com/olegakbarov/585bdf5232f181639487). What do you think?